plotly/plotly_setup_aux/plotlyupdate.m

```diff
@@ -1,365 +1 @@
-                
-                if verbose
-                    fprintf('Done! \n');
-                end
-                
-            catch exception
-                fprintf(['\n\n An error occured while attempting to remove',...
-                    ' the \n temporary update directory. Please remove manually.\n\n']);
-            end
-        end
-        
-        %----successful update----%
-        if success
-            if exist(fullfile(matlabroot,'toolbox','plotly'),'dir')
-                fprintf('\n**************************************************\n');
-                fprintf(['[UPDATE SUCCESSFUL] visit: https://plot.ly/matlab \n',...
-                    'or contact: chuck@plot.ly for further information.\n'...
-                    'Please restart your MATLAB session so that the new\n',...
-                    'Plotly API MATLAB Libary scripts can be recognized.\n']);
-                fprintf('**************************************************\n\n');
-            else
-                fprintf('\n**************************************************\n');
-                fprintf(['[UPDATE SUCCESSFUL] visit: https://plot.ly/matlab \n',...
-                    'or contact: chuck@plot.ly for further information.\n']);
-                fprintf('**************************************************\n\n');
-            end   
-        else
-            fprintf('\n***************************************************\n');
-            fprintf(['[UPDATE UNSUCCESSFUL] visit: https://plot.ly/matlab \n',...
-                'or contact: chuck@plot.ly for further information. \n']);
-            fprintf('***************************************************\n\n');
-        end
-        
-    else %check
-        
-        fprintf(['\nYou are about to update your Plotly API MATLAB Library v.' pvLocal ', which will\n',...
-            'overwrite modifications made to the Plotly scripts in the following directories:\n\n']);
-        
-        % explicitly output directories
-        for d = 1:length(plotlyDirs)
-            fprintf([plotlyDirs{d} '\n']);
-        end
-        
-        overwrite = input('\nProceed with update (y/n) ? : ','s');
-        
-        if(strcmpi(overwrite,'y'))
-            if verbose
-                if nargout
-                    plotlyupdate('verbose','nocheck');
-                else
-                    plotlyupdate('verbose','nocheck');
-                end
-            else
-                if nargout
-                    plotlyupdate('nocheck');
-                else
-                    plotlyupdate('nocheck');
-                end
-            end
-        else
-            fprintf('\n***********************************************\n');
-            fprintf(['[UPDATE ABORTED] visit: https://plot.ly/matlab \n',...
-                'or contact: chuck@plot.ly for more information. \n']);
-            fprintf('***********************************************\n\n');
-        end
-    end
-end
```

Comment:
Basically we now need to handle updating the Matlab-api and Matlab-online versions of the wrapper. So we need to check if the user downloaded the latest version (with the name change) and handle updating that one appropriately. I think going forward we should modify the update script to only update the version of the wrapper located in the MATLAB toolbox (put there after running plotlysetup).